python/lsst/ctrl/bps/drivers.py

```diff
@@ -380,6 +380,30 @@ def submit_driver(config_file, **kwargs):
         "not accurately reflect actual memory usage by the bps process."
     )
 
+    remote_build = {}
+    config = BpsConfig(config_file, BPS_SEARCH_ORDER)
+    _, remote_build = config.search("remoteBuild", opt={"default": {}})
+    if remote_build:
+        if config["wmsServiceClass"] == "lsst.ctrl.bps.panda.PanDAService":
```

Comment:
@MichelleGower Comments? Adding anything to bps that wants to know about a plugin seems wrong.

Reply:
I haven't had a chance to look at this yet other than this one line.  What @timj says is correct.  There shouldn't be a need to write WMS specific code inside ctrl_bps.  If we're wanting to do all of the bps submission stages remotely or let's make that more generic to inside a compute job (even a local job), then we should change the bps code generically to do this.

Reply:
Yes I agree. At the time when we worked on the remote submission, it's not a generic operation for other WMS, so we tried to keep other WMS intact but only apply the remote submission to PanDA. Basically we need bps has a way to stop generating quantum graph and execution bulter at the stage of transform when it's a remote submission.